boards/common/nrf52/Makefile.include

```diff
@@ -11,8 +11,12 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 # because BOARDs Makefile.include are included before Makefile.dep
 # This HACK can be removed either when Makefile.include is included after
 # Makefile.features (see #9913) or if 'nordic_softdevice_ble' is deprecated.
+#
+# Declaring that this hack is being run to allow code in Makefile.dep that
+# depends on the regular sequence of events to skip this early round
+EARLY_BOARD_MAKEFILE_DEP_HACK=1
```

Comment:
`feather-nrf52840` doesn't need that hack.

Reply:
Yeah, that was only needed with the DEFAULT_MODULE construction (where the line below the change, the workaround for some softdevice issue, had an early include of the .dep file that, upset the rest of the usual include dance).